lesson_2/home_work2_6.py

```diff
@@ -0,0 +1,4 @@
+group_of_people = ["John", "Rebecca", "Alessandro", "Melena", "John"]
+uniq_names = list(set(group_of_people))
```

Comment:
Please read task again. And you find some limitations there: you cannot use set for this task.